test/acceptance/kafo_configure_test.rb

```diff
@@ -131,7 +130,7 @@ module Kafo
       end
 
       it 'must create file' do
-        skip 'Requires Puppet 4.5+ for data in modules' if Puppet::PUPPETVERSION < '4.5.0'
+        skip 'Requires Puppet 4.5+ for data in modules' if Gem::Specification.find_all_by_name('puppet').sort_by(&:version).last.version < Gem::Version.new('4.5.0')
```

Comment:
why searching for all puppet versions? is Puppet::PUPPETVERSION removed in 4.10? I'd expect we'd need to do something like `Gem::Version.new(Puppet::PUPPETVERSION)`